static/templates/work/submit.html

```diff
@@ -0,0 +1,88 @@
+<style>
+.file-upload-area {
+	padding: 20px;
+	border: 4px #CCC dashed;
+	text-align: center;
+	font-size: xx-large;
+	color: #CCC;
+	margin-bottom: 20px;
+	min-height: 10%;
+	width: 100%;
+	vertical-align: middle;
+}
+.file-upload-area input[type="file"] {
+	display: none;
+	width: 100%;
+	min-height: 100px;
+}
+.file-upload-dragover {
+	background-color: #F9F9F9;
+	opacity: 0.8;
+}
+
+.table {
+	width: 100%;
+}
+</style>
+
+<md-content class="transparent-background" flex="100" ng-controller="macroWorkSubmitController">
```

Comment:
this is better done with the controller as syntax we use elsewhere "MacroWorkSubmitController as vm"